dusty/commands/setup.py

```diff
@@ -84,6 +85,13 @@ def _get_boot2docker_vm_size():
     else:
         return _get_raw_input('Please input the number of megabytes to allocate to the vm: ')
 
+def _check_ssh_add():
+    output = ''
+    try:
+        check_call_demoted(['sh', '-c', 'ssh-add -l'])
```

Comment:
Why do we need a shell here? Also don't we always do Git commands as root? If so, this shouldn't be demoted.